app/annotation/annotation.cjsx

```diff
@@ -65,7 +66,8 @@ Annotation = React.createClass
   componentWillUnmount: ->
     window.removeEventListener 'keydown', @onPressSpaceBar
 
-  zoomImage: (preview) ->
+  zoomImage: (i, preview) ->
+    @setState zoomImageCurrent: i
```

Comment:
Instead of calling `@setState` twice, both can be passed in as a single object with their new values:

`@setState({zoomImageCurrent: i, zoomImageSrc: preview})`